From 5ddda749c60dce834bcd79e8b8d904858319adc0 Mon Sep 17 00:00:00 2001
From: Rupinderjit Singh <rupinderjit.singh@arm.com>
Date: Tue, 7 Feb 2023 09:45:02 +0000
Subject: [PATCH] plat-totalcompute: update TZDRAM_SIZE

For CFG_CORE_SEL2_SPMC, manifest size is increased from 0x1000 to
0x4000 for boot protocol support.

Signed-off-by: Rupinderjit Singh <rupinderjit.singh@arm.com>
Acked-by: Jens Wiklander <jens.wiklander@linaro.org>

Upstream-Status: Backport
Signed-off-by: Jon Mason <jon.mason@arm.com>

---
 core/arch/arm/plat-totalcompute/conf.mk | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/core/arch/arm/plat-totalcompute/conf.mk b/core/arch/arm/plat-totalcompute/conf.mk
index b39ac0f0667f..2f6c0ee1460a 100644
--- a/core/arch/arm/plat-totalcompute/conf.mk
+++ b/core/arch/arm/plat-totalcompute/conf.mk
@@ -32,8 +32,9 @@ ifeq ($(CFG_CORE_SEL1_SPMC),y)
 CFG_TZDRAM_START ?= 0xfd000000
 CFG_TZDRAM_SIZE  ?= 0x02000000
 else ifeq ($(CFG_CORE_SEL2_SPMC),y)
-CFG_TZDRAM_START ?= 0xfd281000
-CFG_TZDRAM_SIZE  ?= 0x01d7f000
+CFG_TZDRAM_START ?= 0xfd284000
+# TZDRAM size 0x1980000 - 0x4000 manifest size
+CFG_TZDRAM_SIZE  ?= 0x0197c000
 else
 CFG_TZDRAM_START ?= 0xff000000
 CFG_TZDRAM_SIZE  ?= 0x01000000
